code/game/machinery/anomaly_pad.dm

```diff
@@ -17,6 +17,11 @@
 	///our base active power use. Decreases with laser efficiency and is * 0.1 for inactive
 	var/base_power = 500
 
+/obj/machinery/anomaly_pad/Initialize()
+	. = ..()
+
+	STOP_PROCESSING(SSmachines, src)
```

Comment:
This will conflict with the standard set by https://github.com/tgstation/tgstation/pull/49307 so whichever gets merged first will have to be accounted by the other.

Reply:
Thanks for the warning, I'll either fix it or warn the other guy if this gets merged first